src/sentry/static/sentry/app/views/settings/components/forms/formPanel.tsx

```diff
@@ -11,7 +11,7 @@ type Props = {
   /**
    * Panel title
    */
-  title?: string;
+  title?: string | object;
```

Comment:
When a prop is a string or react element, we will want to use  `React.ReactNode`

```suggestion
  title?: React.ReactNode;
```